packages/neuron-wallet/src/services/sync/get-blocks.ts

```diff
@@ -46,7 +46,12 @@ export default class GetBlocks {
         const checkTx = new CheckTx(tx, this.url)
         const addresses = await checkTx.check(lockHashes)
         if (addresses.length > 0) {
-          for (const input of tx.inputs!) {
+          const inputs = tx.inputs!
+          for (let i = 0; i < inputs.length; ++i) {
+            if (i === 0) {
```

Comment:
That's the first tx of a block, not the first input of a tx.

Reply:
fixed